ang/crmCaseType/edit.html

```diff
@@ -6,7 +6,7 @@ <h1 crm-page-title>{{caseType.title || ts('New Case Type')}}</h1>
 <form name="editCaseTypeForm" unsaved-warning-form>
 <div class="crm-block crm-form-block crmCaseType">
   <div class="help">
-    {{ts('Use this screen to define or update the Case Roles, Activity Types, and Timelines for a case type.')}} <a href="http://book.civicrm.org/user/current/case-management/setup/" target="_blank">{{ts('Learn more...')}}</a>
```

Comment:
This changes the hostname also, I think we no longer use book.civicrm.org (seems to be a redirect now).